src/components/Overview/Overview.react.js

```diff
@@ -46,16 +73,145 @@ class Overview extends Component {
       isVideoModalOpen: !prevState.isVideoModalOpen,
     }));
   };
-
   componentDidMount() {
     document.body.style.backgroundColor = '#fff';
     // Adding title tag to page
     document.title =
       'SUSI.AI - Open Source Artificial Intelligence for Personal Assistants, Robots, Help Desks and Chatbots.';
     //  Scrolling to top of page when component loads
     scrollToTopAnimation();
+    setInterval(() => this.checkgifActive(), 7000);
+  }
+  checkgifActive() {
```

Comment:
Some changes -
* Change the logic according to single state use.
* Remove console.log
* Use only 1 color for active state for buttons.
* Use switch case instead of multiple if cases.
* Some gifs appear to be broken while switching between examples.